README.md

```diff
@@ -163,6 +163,46 @@ helm tiller run \
 -    --set container.image=eu.gcr.io/census-eq-ci/eq-survey-runner-benchmark:latest
 ```
 
+## Visualising Benchmark Results
+
+You can use the `visualise_results.py` script to visualise benchmark results over time. 
+
+### Pre-Requisites
+
+You need to be authenticated with GCP in order to download the benchmark results. To do this:
+
+Log In with gcloud auth using:
+
+`gcloud auth login`
+
+`gcloud auth application-default login`
```

Comment:
I had to run `gcloud auth application-default login --project <valid_project_id>` or I got this error:

```
google.api_core.exceptions.BadRequest: 400 GET https://storage.googleapis.com/storage/v1/b/eq-daily-performance-test-benchmark-outputs?projection=noAcl: User project specified in the request is invalid.
```

Reply:
So it looks like I was getting errors because the project I had set in the gcloud config was invalid (an old project). This can be set by:
- adding it to the config using - `gcloud config set project <project_id>`, or 
- passing it to the gcloud auth login command - `gcloud auth login --project <project_id>`

This needs to be set before running the `gcloud auth application-default login` command. I think we should change `gcloud auth login` to `gcloud auth login --project <project_id>` to prevent people from getting this error.